eplaunch/interface/crossplatform.py

```diff
@@ -0,0 +1,20 @@
+import platform
```

Comment:
A file to unify cross platform specifics, mostly a set of predefined enums plus a single worker to get the current platform.